client/src/app-components/page-dropdown.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import DropDown from 'core-components/drop-down';
+
+class PageDropdown extends React.Component {
```

Comment:
Let's call it `PageSizeDropdown`, no fear of making names a bit longer, specially here where it disambiguates between two meanings.